src/SAMLTrace.js

```diff
@@ -611,6 +611,15 @@ SAMLTrace.TraceWindow.prototype = {
       type = type.substr(0, i);
     }
     type = type.toLowerCase().trim();
```

Comment:
Interested in some additional optimization?
One could shorten the code right from the beginning of the function like this:

```javascript
const contentTypeHeader = request.responseHeaders.find(header => header.name.toLowerCase() === 'content-type');
if (contentTypeHeader === null) {
  return true;
}
const type = contentTypeHeader.value.split(';')[0].toLowerCase();
const basetype = type.split('/')[0];
``` 

* `.find` yields only a single entry, instead of `.filter`
* The if can be replaced by direct access to the first index of a split call. This is safe even if there's no semi-colon with the string.

Reply:
Thanks! Your suggestion dropped the `trim()`, probably not intentionally, so I've added that but used it verbatim otherwise.